cengage/react-magma
# fix/simple pagination

Issue: # [991](https://github.com/cengage/react-magma/issues/991)

## What I did
- Fixed previous and next button navigation via keyboard.
- Tweaked VoiceOver for reading the pages title and count.
- Tweaks to responsive behavior.
- Fixed `onPageChange` function to properly retain current page.

## Screenshots (if appropriate):
![hooray](https://user-images.githubusercontent.com/77400920/227005877-40286e31-fc2e-4aa4-8de1-e6efa17eac0c.png)

## Checklist 

## How to test

- Confirm keyboard navigation works appropriately with previous and next buttons.
- Verify "Page number ##, of ## pages" is read by VoiceOver.
- Check responsive behavior with different screen widths.
